### DISCUSSION TITLE
Shortcut to create page in Publii

### DISCUSSION THREAD

**Discussion by PapsOu** [GetPublii/Publii#1819] [General]
Hello,

Is there any custom protocol handler that can handle shortcut pointing to the publii desktop application ?

My need is pretty simple : I want to generate a link from our web application of that form :

```
publii://createPage?site=<web_site_id>&slug=<target_slug_page>
```

*where `<web_site_id>` is the target site name and `<target_slug_page>` is the pre-fill slug page (SEO section) when creating a new page.*

According to https://unix.stackexchange.com/questions/497146/create-a-custom-url-protocol-handler, i would make that kind of custom protocol handler to handle this specific link.

Do you know if the desktop application (electron-based) can handle that kind of shortcut ?

**Comment by dziudek** [comment]
@PapsOu - as I see it is possible in Electron apps and quite simple to implement - I have added it for v.0.47, because I have to prepare list of such supported operations and implement it. 

If you have any other propositions - feel free to describe them here :)
